packages/create-app/src/cli.ts

```diff
@@ -90,7 +97,7 @@ export async function cli(args: string[] = process.argv) {
 
   const parsed: CliArgs = base.parseSync();
   const project: string = await promptProject(parsed);
-  const overwrite: boolean = await promptOverwrite(parsed, project);
+  const overwrite: boolean = await promptOverwrite({ ...parsed, project });
```

Comment:
I think its more clear to pass the actual expected object here?
```suggestion
  const overwrite: boolean = await promptOverwrite({ overwrite: parsed.overwrite, project });
```

Reply:
Hmm I actually feel it's more consistent with the other prompts which is saying you can have access to all of the parsed defaults but the project is replaced with the final name from the previous prompt